src/query/settings/src/settings_default.rs

```diff
@@ -625,7 +625,7 @@ impl DefaultSettings {
                     range: Some(SettingRange::Numeric(0..=1)),
                 }),
                 ("enable_experimental_aggregate_hashtable", DefaultSettingValue {
-                        value: UserSettingValue::UInt64(0),
+                        value: UserSettingValue::UInt64(1),
```

Comment:
Is ready default to use the new aggregate hashtable? @sundy-li

Reply:
It's set to 1 only to check the ci tests.

Reply:
It will not be enabled after full tests (perf test, ci tests).

 Now we change to 1 just to pass the ci tests.
 It will be disabled before merge.

Reply:
Add a note to the PR summary:
<img width="821" alt="image" src="https://github.com/datafuselabs/databend/assets/172204/c9550c06-45b2-46e7-a71b-24f261269920">

@JackTan25 It's not affected your scope: `merge into`,  and this settings is a temporary enable.